src/payment/strategies/adyenv2/adyenv2-payment-strategy.spec.ts

```diff
@@ -441,20 +402,10 @@ describe('AdyenV2PaymentStrategy', () => {
                 ThreeDS2ComponentType.ThreeDS2Challenge,
                 componentOptions
             );
-            expect(adyenCheckout.create).toHaveBeenCalledTimes(3);
+            expect(adyenCheckout.create).toHaveBeenCalledTimes(4);
```

Comment:
Not sure why you're asserting that a method has been called 4 times. Don't you think it's a bit random? I think you're testing a state change but have not reset the spy before the state change. If you do that, you can probably write a more precise assertion and make the test a lot easier to read.

Reply:
In fact we are calling (now) `create` method 4 times.
1.- Payment component (i.e. scheme, ideal, etc.)
2.- treeDS2Component (fingerprint)
3.- treeDS2Component (challenge)
4.- customFields component (this is the one been added)